pyomo/contrib/pynumero/algorithms/solvers/tests/test_cyipopt_interfaces.py

```diff
@@ -156,9 +156,9 @@ def test_model1_CyIpoptNLP_scaling(self):
 
         cynlp = CyIpoptNLP(PyomoNLP(m))
         obj_scaling, x_scaling, g_scaling = cynlp.scaling_factors()
-        self.assertTrue(obj_scaling is None)
-        self.assertTrue(x_scaling is None)
-        self.assertTrue(g_scaling is None)
+        self.assertTrue(obj_scaling == 1.0)
+        self.assertTrue((x_scaling == 1.0).all())
+        self.assertTrue((g_scaling == 1.0).all())
```

Comment:
Not sure how others feel about this, but I would prefer to maintain backwards compatibility here. I care less about the `CyIpoptNLP.scaling_factors` than maintaining consistency of `PyomoNLP.get_primals_scaling`. Somebody could very well be using `PyomoNLP.get_primals_scaling() is None` to check whether scaling is used.